packages/jbrowse-plugin-apollo/src/ApolloRenderer/components/ApolloRendering.tsx

```diff
@@ -63,6 +65,22 @@ function ApolloRendering(props: ApolloRenderingProps) {
     // @ts-ignore
     Array.from(a.values()).map((f) => getSnapshot(f)),
   )
+
+  useEffect(() => {
+    const { internetAccounts } = getRoot(session) as AppRootModel
+    const apolloInternetAccount = internetAccounts.find(
+      (ia) => ia.type === 'ApolloInternetAccount',
+    ) as ApolloInternetAccountModel | undefined
+    if (!apolloInternetAccount) {
+      throw new Error('No Apollo internet account found')
+    }
```

Comment:
We have to make sure the internetAccount is the correct one, instead of the first one. You'll need to use the assemblyName from the region and the assemblyManager to get the internetAccountConfigId for the assembly and use that to find the correct account. An example of that is here: https://github.com/GMOD/Apollo3/blob/0ee1d3f9781b468215483fa59ff108827c44c112/packages/apollo-shared/src/BackendDrivers/CollaborationServerDriver.ts#L21-L40